docs/file-reference.md

```diff
@@ -362,6 +367,30 @@ of `service`.
 `endpoint` the way it is defined is can actually can be divided into
 two parts the `URL` and `Path`, it is delimited by a forward slash.
 
+#### portMappings
+```yaml
+portMappings:
+- 8081:81/UDP
+```
+
+`portMappings` is an added field to ServiceSpec.
+This lets us set the port, targetPort and the protocol for a service in a single line. This is parsed and converted to a Kubernetes ServicePort object.
+
+`portMappings` is an array of `port:targetPort/protocol` definitions, so the syntax looks like -
+
+```yaml
+portMappings:
+- <port:targetPort/protocol>
+- <port:targetPort/protocol>
+```
+
+The only field mandatory here is "port".
```

Comment:
Can you specify that you are talking about `port` part in the `portMappings`. When I read this for the first time I thought you are talking about `port` in Service.


How about this?
'The only mandatory part of the `portMappings` is `port`'

Reply:
makes sense, updated